### DISCUSSION TITLE
2023-12-30-30

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#163] [General]
# 我居然 30 岁了 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Thought过完今晚我居然就满 30 岁了，一直很无耻的感觉自己才 24 岁，虽然在这之前，我更无耻的认为自己才 18 岁，真不敢想象时间怎么过得这么快，年纪好像是一个从山顶滚下来的轮子，随着时间拉长越滚越快。虽然经历越来越丰富，但一年的时间长度感知明显比小时候得短很多，还是会有些惋惜和遗憾。

https://tw93.fun/2023-12-30/30.html

**Comment by yexia553** [comment]
是呀，30了

**Reply by tw93** [reply]
一转眼就....

**Comment by JoinWorld** [comment]
知道你昵称的由来了，同93，也30了，刚刚起步，希望未来越来越好

**Reply by tw93** [reply]
哈哈 是的，谢谢！

**Reply by Jie-Huangi** [reply]
哈哈，之前还在想为什么要93呢？原来是这样。

**Comment by sangyuxiaowu** [comment]
评论一下，看看是不是sang93 😂

**Comment by DianNaozi** [comment]
我现在24岁在一个小企业工作了8个月，目前贼焦虑

**Reply by tw93** [reply]
还这么年轻，焦虑啥，想明白生活的意思是什么就好啦

**Reply by Jie-Huangi** [reply]
哈哈，就是没想明白，才焦虑的哦。博主可以专门讲一期生活的意义就好啦。

**Comment by lighterEB** [comment]
是啊，时间真快一上午就过去了，又该吃午饭了 ：）

**Reply by tw93** [reply]
哈哈哈哈

**Comment by ZyMousai** [comment]
特意登陆膜拜一下大佬，下载了妙言准备试试。

**Reply by tw93** [reply]
谢谢喜欢

**Comment by linkgod** [comment]
最近找了个00后的实习生，突然明白原来当初舒克哥看我的样子，就是我现在看他们的样子

**Comment by liulei53** [comment]
我们读的书还挺有相似度的，置身事外，马斯克传，为了和雷布斯做对比，专门看了雷布斯的小米创业十年。会一直关注你，很棒。

**Reply by tw93** [reply]
🤛🤛

**Comment by lznauy** [comment]
我00年的都已经快26了，还一事无成，也挺焦虑，现在新人都是02、03的，感觉时间真快啊
